tools/licenser_postprocess.py

```diff
@@ -0,0 +1,56 @@
+# this script is used to generate licenses-related stuff in docs
+
+from os import system, getcwd, listdir, chmod, rename
+from os.path import dirname, basename
+
+# where we work
+Dir = dirname(__file__) + "/../web/"
+
+# copy licenses from customlicenses
+Licenses = listdir(Dir + "customlicenses")
+for License in Licenses:
+	system("cp " + Dir + "customlicenses/" + License + " " + Dir + "docs/licenses/" + License)
```

Comment:
Let's not revert to using system commands.

We can use [`shutil.copyFile`](https://docs.python.org/3/library/shutil.html#shutil.copyfile) for this.